net/yolov2/test.py

```diff
@@ -78,14 +79,27 @@ def postprocess(self, net_out, im, save = True):
 			if top   < 0    :   top = 0
 			if bot   > h - 1:   bot = h - 1
 			thick = int((h+w)/300)
-			cv2.rectangle(imgcv, 
-				(left, top), (right, bot), 
+			cv2.rectangle(imgcv,
+				(left, top), (right, bot),
 				colors[max_indx], thick)
 			mess = '{}'.format(label)
-			cv2.putText(imgcv, mess, (left, top - 12), 
+			cv2.putText(imgcv, mess, (left, top - 12),
 				0, 1e-3 * h, colors[max_indx],thick//3)
+			if self.FLAGS.json:
```

Comment:
Please bring this `if` block to before line 82 and put a `continue` at the end of the block